source/game.stationmap.bmx

```diff
@@ -4564,21 +4564,22 @@ Type TStationAntenna Extends TStationBase {_exposeToLua="selected"}
 			'section specific costs for bought land + bureaucracy costs
 			buyPrice :+ section.GetPropertyAquisitionCosts(TVTStationType.ANTENNA)
 			'section government costs, changes over time (dynamic reach)
-			buyPrice :+ 0.20 * GetReceivers()
+			buyPrice :+ 0.35 * GetReceivers()
 			'government sympathy adjustments (-10% to +10%)
 			'price :+ 0.1 * (-1 + 2*channelSympathy) * price
 			buyPrice :* 1.0 + (0.1 * (1 - 2*channelSympathy))
 
 			'fixed construction costs
 			'building costs for "hardware" (more expensive than sat/cable)
-			buyPrice :+ 0.20 * GetPopulation()
+			'TODO check - reduced!! 0.2*receivers+0.2*popultation causes implausible prices for later years population big, but receivers very small
+			If includePopulation Then buyPrice :+ 0.05 * GetPopulation()
```

Comment:
Die Preise sollten zwar spaeter in Relation "geringer" ausfallen (guenstigere Bauweisen, effizientere Technik). Aber dennoch braucht eine grosse "Sendereichweite" (Flaeche):
- teure Technik
- spezielle Aufbauten (hohe Masten, ...)

Nahezu "nur" durch die Bevoelkerungsdichte sollte der Preis nicht definiert sein.
Allerdings waere hier zu ueberlegen, ob man nicht statt "getPopulation" einen Referenzgeldwert nimmt und den Radius als Faktor einbezieht (bei Antennen). Denn der Radius ist ja sozusagen die "Sendeleistung".

Reply:
Das (nachvollziehbare) Argument ließe sich auf die laufenden Kosten übertragen - größerer Radius=höhere Sendeleistung (mehr Strom) unabhängig von Reichweite/Bevölkerung.

Sollte also das includePopulation-Flag wieder raus und statt GetPopulation() ein radiusabhängiger Anteil rein?
Aber auch da muss man ja aufpassen, da gerade in den späten Startjahren ja ein großer Radius einer kleinen Reichweite gegenübersteht ...

Reply:
Theoretisch sind ja "frueher" denke ich Antennen an sich "teurer" gewesen.
Heute ist die Teuerung ja nicht in der Technik direkt zu suchen, sondern in steigenden Materialkosten, Immobilienpreise, Techniker/Personalkosten.

Inwieweit man das am Ende als "konstant" betrachten kann, weiss ich nicht.

Der Anteil "radiusbezogener" Kosten sollte evtl nicht-linear sein (`x hoch 0.xyz`) damit spaetere Spiele (kurz vor Kabelnetz-Alternative) nicht zu teure Antennen haben (also die zweite Antenne waere dann ja vlt zu teuer).

Aber ja, gern das includePopulation raus und einen anderen Faktor rein.

Reply:
Ich bin nicht ohne den Schalter ausgekommen. Entweder ändert sich der Kaufpreis nicht signifikant oder die laufenden Kosten haben ein zu breites Spektrum (zwischen 100 und 1300 pro 1K Zuschauer).
Durch die extrem verschiedenen Bevölkerungsdichten können auch große Antennenreichweiten z.T. sehr wenige Zuschauer erreichen (30000). Wenn dann der zuschauerunabhängige Anteil bei den laufenden Kosten zu groß wird, gehen die Preise auseinander.

Bei Kabel- und Satellitenverträgen ist der Bereich nicht so extrem.

Reply:
Aber so ist es ja an sich auch in der Realitaet ... sprich man versucht mit den geringsten Kosten die meisten Zuschauer zu erreichen.

Welchen "Vorteil" sehe ich da? Die Sender/Spieler werden sich anfaenglich durchaus auf einige "Zonen" auf der Karte konzentrieren. Sprich "Marktgesetze" kommen automatisch zur Geltung.

Weiterhin heisst dies auch, dass Brachland-Zonen (Brandenburg und Co.) mit wenig Bevoelkerungsdichte "teurer" zu besenden sind. Hier waere dann sogar interessant (-> TODO, neues Issue, falls wirklich interessant) neue Achievements einzubauen. Und zwar "> 95% des Bundeslands abgedeckt". Die Belohnung kann in Relation zur "Pixelflaeche/Bevoelkerungsdichte" ausfallen (muesste man beim Laden der Senderkarte einfach ermitteln, nicht nur die Bevoelkerungszahl sondern auch "Pixel").

Reply:
Wenn wir die Bevoelkerungsdichte eines Bundeslands haben ("allgemein") und die Bevoelkerungsdichte der Antenne (die haben wir ja durch "Bevoelkerung" und der Flaeche berechnet durch den Radius) koennen wir auch Rueckschluesse auf die "Bodenbegebenheiten" ziehen.

Hohe Bevoelkerungsdichte im Areal: "bebaut"
Niedrige Bevoelkerungsdichte: unbebaut, guenstige Baukosten/Immobilienpreise, Infrastrukturkosten vtl hoeher (Kabel verlegen etc)

Das koennte man meines Erachtens nach sogar noch interessanter gestalten, in dem man die Bevoelkerung einer Antenne mit "radius = 2-3" macht ("Bauland"-Flaeche und Umgebung) und dies fuer Baukosten relevant macht.

Das erzeugt den wunderbaren Effekt, dass ein paar "Pixel weiter links" evtl die gleiche Reichweite erreicht werden kann, aber Bau- und Unterhaltskosten spuerbar anders sind.

Je weiter ich jetzt hier drueber schreiben wollen wuerde, desto mehr wird klar, dass es ein eigenes Diskussionsthema/Issue sein sollte.

Reply:
Die Baukosten pro Bundesland waren schon im Altcode berücksichtigt (basePrice :+ section.GetPropertyAquisitionCosts(TVTStationType.ANTENNA)).

Bei allen Vorschlägen, wie man es realistisch/interessanter machen kann, sehe ich aktuell keine Lösung für das grundsätzliche Problem, dass das Verhältnis Antennen/Kabel/Satellit in späten Spieljahren eine ziemliche Unwucht hat; insb. wenn man noch in Kauf nimmt (oder explizit fordert), dass die laufenden Kosten dermaßen unterschiedlich sein können.

Bei einem Startjahr 2005 kommt man auf ca 10 Mio Zuschauer durch Antennen. Um mit dieser Zuschauerzahl (bei z.T. sehr hohen laufenden Kosten) genug anzusparen, um einen lukrativen (hohe Reichweite) Kabelvertrag zu unterschreiben, dauert seine Zeit. Wenn man dann aber mal die Verträge hat (geringe laufende Kosten im Vergleich zu den Antennen), geht es mit dem Geld ab dann plötzlich sehr schnell. 

D.h. zunächst quält man sich ewig ab und dann gibt es überhaupt keinen Widerstand mehr. Oder um beim Vergleich der Lua-Spieler zu bleiben: einer hängt bei 10 Mio fest, ein anderer ist schon bei 70 Mio.

Dem würde ich gerne mit eher niedrigen laufenden Kosten für Antennen (insb. bei geringer Reichweite) entgegenwirken. In frühen Jahren dauert es, weil man wegen des kleinen Radius viele Antennen bauen (und Genehmigungen einholen) muss. In späteren Jahren kann man "relativ schnell" die gesamte Fläche mit Antennen vollstellen, erreicht aber vergleichsweise wenige Leute, hat also weniger Einnahmen.

Wärst Du damit einverstanden, dass an den "Festpreisfaktor" noch ein TODO drankommt, aber die grundsätzliche Unterscheidung des Basispreis für Kauf und laufende Kosten so bleibt?